### ISSUE TITLE
Ensure the language of a document is set

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3467]
2024 OCIO Report

Instances: 4
Occurrence: 1%
Severity: 3
Priority: 2.4

Location: https://revenuedata.doi.gov/downloads/production/

Note: This HTML does not have a lang attribute
